mondey_backend/src/mondey_backend/models/questions.py

```diff
@@ -61,6 +63,7 @@ class UserQuestion(Question, table=True):
     answers: Mapped[list[UserAnswer]] = back_populates(
         "question", cascade="all, delete-orphan"
     )
+    visibility: bool = False
```

Comment:
I'm not sure if we need to have visibility for individual texts - isn't it enough that the question is either visible or not?

Reply:
This comment is wrong (sorry I got fooled by github's code folding), but I think we can still remove visibility since this inherits from Question which has it already